cfme/tests/v2v/test_v2v_cancel_migrations.py

```diff
@@ -4,97 +4,92 @@
 import fauxfactory
 import pytest
 
+from cfme import test_requirements
+from cfme.cloud.provider.openstack import OpenStackProvider
 from cfme.fixtures.provider import rhel7_minimal
 from cfme.infrastructure.provider.rhevm import RHEVMProvider
 from cfme.infrastructure.provider.virtualcenter import VMwareProvider
 from cfme.markers.env_markers.provider import ONE_PER_TYPE
 from cfme.markers.env_markers.provider import ONE_PER_VERSION
-from cfme.utils.appliance.implementations.ui import navigator
-from cfme.utils.log import logger
 from cfme.utils.wait import wait_for
 
 pytestmark = [
     pytest.mark.provider(
-        classes=[RHEVMProvider],
+        classes=[RHEVMProvider, OpenStackProvider],
         selector=ONE_PER_VERSION,
-        required_flags=['v2v'],
-        scope="module"
+        required_flags=["v2v"],
+        scope="module",
     ),
     pytest.mark.provider(
         classes=[VMwareProvider],
         selector=ONE_PER_TYPE,
-        fixture_name='source_provider',
-        required_flags=['v2v'],
-        scope="module"
-    )
+        required_flags=["v2v"],
+        fixture_name="source_provider",
+        scope="module",
+    ),
+    pytest.mark.usefixtures("v2v_provider_setup")
 ]
 
 
-@pytest.mark.parametrize('form_data_multiple_vm_obj_single_datastore', [['nfs', 'nfs',
```

Comment:
@sshveta  can you please add tier, we execute it on 'High' priority since we have seen multiple failures in this area.
```@pytest.mark.tier(1)``` will be appropriate I guess.

Reply:
:+1:

Reply:
Tier is not the same as importance, tier indicates the type of test. `caseimportance` is set in polarion metadata to Critical/High/Medium/Low.

Reply:
These are pretty much all tier 2 or tier 3 tests

Reply:
oh! I was thinking tier 1 stands for important tests , got you, thanks.